libraries/standard/mqtt/src/mqtt.c

```diff
@@ -144,11 +158,34 @@ static MQTTStatus_t handleIncomingPublish( MQTTContext_t * const pContext,
  *
  * @param[in] pContext MQTT Connection context.
  * @param[in] pIncomingPacket Incoming packet.
+ * @param[in] manageKeepAlive Flag indicating if PINGRESPs should not be given
```

Comment:
(Minor) To make the parameter more meaningful, I suggest renaming it to something like `notifyAppOfPingResponse`..

Reply:
I chose manageKeepAlive since I thought it was more meaningful to indicate whether or not keep alive is being handled by the library, rather than indicating if the app should be notified of pingresps. The latter implies the question "why shouldn't the app be notified?" - and the answer to which is that the library is managing keep alive. I can update the param description to better reflect this